src/scripts/build_all_repos.sh

```diff
@@ -556,7 +560,7 @@ function has_mvn () {
                 error 82 "has_mvn fetch mvn epel repo $EPEL_MVN_REPO failed"
             fi
 
-            $SUDO yum makecache "$DISABLEREPOSFULL" "$ENABLEREPOSFULL"
+            $SUDO yum $YUMMAKECACHE
```

Comment:
Add this on the previous line to disable the quoting check
```bash
# shellcheck disable=SC2068
```

Reply:
Maybe this is used enough that it would be cleaner to move the yum cache handling to a function and embed the do or do not logic there instead.